plugins/pulp_rpm/plugins/distributors/export_distributor/generate_iso.py

```diff
@@ -87,9 +88,26 @@ def _make_iso(file_list, target_dir, output_dir, filename):
     if status != 0:
         log.error("Error creating iso %s; status code: %d; output: %s" % (file_path, status, out))
     else:
+        _add_digest_file(file_path)
         log.info('Successfully created iso %s' % file_path)
 
 
+def _add_digest_file(file_path):
+    exportedISO = open(file_path, "rb")
+    digestFile = open(file_path + ".DIGEST", "w+")
+    hashers = {}
+    for hasher in hashlib.algorithms_guaranteed:
+        hashers[hasher] = getattr(hashlib, hasher)()
+        chunk = 0
+        while chunk != b'':
```

Comment:
The content of the .DIGEST file is incorrect. Only one of the checksums matches the sum calculated on command line. 

You should un-indent the code that reads the file. Line 104 needs to be inside a loop that looks exactly like line 99.